projects/metis/src/app/dataset/workflow/workflow.component.spec.ts

```diff
@@ -268,6 +301,7 @@ describe('WorkflowComponent', () => {
       expect(component.getSaveNotification()!.content).toBe('en:workflowSaved');
       expect(component.getSaveNotification()).toEqual(component.notification);
       expect(workflows.createWorkflowForDataset).toHaveBeenCalled();
+      tick(1);
```

Comment:
(this additional tick was needed because of the extra delay put into the queue by the mock workflow service)